src/styles/sidebar/components/share-annotations-panel.scss

```diff
@@ -0,0 +1,53 @@
+.share-annotations-panel {
+  color: $grey-5;
+
+  .share-annotations-panel__intro {
```

Comment:
Nesting these selectors generates descendant selectors (`.share-annotations-panel .share-annotations-panel__intro`) with increased specificity, which can lead to unexpected conflicts. The purpose of including the full component/"block" name in each class is to avoid the need to do this.

That is why our SASS files are generally structured like this:

```scss
.some-component {
}

.some-component__element {
}
```

If you want to avoid writing out the component name in full multiple times, you can also do this:

```scss
.some-component {
  &__element {
    ...
  }
}
```

This will generate:

```css
.some-component__element {
 ...
}
```

Reply:
Thanks! I meant to do the latter but did not do it correctly.